src/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer.c

```diff
@@ -27,10 +27,7 @@ ActorProfile Npc_Engineer_Profile = {
 };
 
 struct_809AEFA4 D_809AEFA4_jp = {
-    func_809AEF28_jp,
-    func_809AEF54_jp,
-    4,
-    0,
+    func_809AEF28_jp, func_809AEF54_jp, 4, 0, 0, 0,
```

Comment:
```suggestion
    aNEG_actor_move, aNEG_actor_draw, 4, 0, 0, 0,
```

Reply:
oh also i hadnt noticed that idk maybe format squished the whole struct into a single line, is that ok?

Reply:
Its sort of up to whatever looks best. a lot of times you can play with adding or removing the final comma in the struct which causes the format script to do different things.